### DISCUSSION TITLE
为什么3.x版我的相关注解在idea里都会爆红（但是不影响运行但没有代码提示）而2.x版不会有这样的问题

### DISCUSSION THREAD

**Discussion by kongke7** [simple-robot/Discussions#155] [Q&A] [answered]
### 涉及的编程语言

Java

### 涉及的组件库

mirai (QQ)

### 疑问描述

为什么3.x版我的相关注解在idea里都会爆红（但是不影响运行但没有代码提示）而2.x版不会有这样的问题

**Comment by ForteScarlet** [comment, marked as answer]
https://simple-robot-library.github.io/simbot3-website/faq/%E5%8C%85%E5%BC%95%E7%94%A8%E5%BC%82%E5%B8%B8/

**Comment by kongke7** [comment]
谢谢，已解决
&nbsp;


??
***@***.***



&nbsp;




------------------&nbsp;Original&nbsp;------------------
From: ***@***.***&gt;; 
Date: 2023年7月4日(星期二) 下午3:46
To: ***@***.***&gt;; 
Cc: ***@***.***&gt;; ***@***.***&gt;; 
Subject: Re: [simple-robot/Discussions]为什么3.x版我的相关注解在idea里都会爆红（但是不影响运行但没有代码提示）而2.x版不会有这样的问题 (Discussion #155)






https://simbot.forte.love/faq/%E5%8C%85%E5%BC%95%E7%94%A8%E5%BC%82%E5%B8%B8/

—
直接回复此邮件，在GitHub上查看，或者取消订阅.
***@***.***与&gt;.

**Reply by Xbai11** [reply]
请问如何解决的我遇到了类似的错误

**Reply by Xbai11** [reply]
作者解答的url已失效

**Reply by kongke7** [reply]
我的是禁用kotlin插件就像

**Reply by Xbai11** [reply]
我也解决的应该是kotlin版本太低了导致的我的idea是2020的自带的版本是1.4的升级后也只能到1.6，文件要求是1.8的我认为应该是这个原因，不知道他真实原因是不是这个

**Reply by kongke7** [reply]
🤔 升级完了好像要把插件禁用掉
